ui/app/AppLayouts/Communities/views/ControlNodeOfflineCommunityView.qml

```diff
@@ -0,0 +1,123 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtGraphicalEffects 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import AppLayouts.Communities.panels 1.0
+import AppLayouts.Chat.views 1.0
+
+import StatusQ.Layout 0.1
+
+import utils 1.0
+import shared.popups 1.0
+
+StatusSectionLayout {
+    id: root
+
+    // General properties:
+    property bool amISectionAdmin: false
+    property bool openCreateChat: false
+    property string name
+    property string communityDesc
+    property color color
+    property string channelName
+    property string channelDesc
+    property bool joinCommunity: true // Otherwise it means join channel action
```

Comment:
I think a couple of those properties are not needed?

Reply:
cleaned up